### DISCUSSION TITLE
Game option for more than 2 preludes and more than 1 corporation

### DISCUSSION THREAD

**Discussion by HenrikSteenbock** [bafolts/terraforming-mars#4820] [Ideas]
Is it possible without much of a hassle to add a game option to select a custom number of preludes/corporations to play with? I'm sure my group is not the only which may like that. Could someone kindly point out where I'd need to make changes to the code to make that happen myself on a local copy of the game?

**Comment by kberg** [comment]
What do you mean by "custom number"? Preludes to choose from? Preludes to play?

**Reply by HenrikSteenbock** [reply]
Primarily, number of preludes and corporations to play. Ideally, also the number to choose from.

**Comment by kberg** [comment]
You'll need to change SelectInitialCards.ts, and SelectInitialCards.vue. You'll probably need to change some code in playCorporationCard. `initialActions` is kind of set up for this, but not for having multiple corporations as part of the initial draw.
